Makefile

```diff
@@ -54,7 +54,8 @@ doc-deps: ## Build the documentation for the local package and all dependencies.
 
 .PHONY: gen-rpc-doc
 gen-rpc-doc:  ## Generate rpc documentation
-	rm -f target/doc/ckb_rpc/module/trait.*.html
+
+	rm -f ${CARGO_TARGET_DIR}/doc/ckb_rpc/module/trait.*.html
```

Comment:
Could not work as expected, because:
https://github.com/nervosnetwork/ckb/blob/28272b0197151c20a27b0797f0e2510a7b685c4f/Makefile#L10

Reply:
It's better to use absolute path for `CARGO_TARGET_DIR`.

```
CARGO_TARGET_DIR ?= $(shell pwd)/target
```

Reply:
If change to `CARGO_TARGET_DIR ?= $(shell pwd)/target`, should be refine run.sh at this time. Let me have a look on it.